windowsforms/matching-game/net5-windows/vb/MatchingGame.Logic/My Project/AssemblyInfo.vb

```diff
@@ -0,0 +1,35 @@
+﻿Imports System
```

Comment:
Is AssemblyInfo needed?

Reply:
Yes, the migration article references this code and part of migration is keeping your assembly info.